azure-cron.yml

```diff
@@ -68,6 +68,7 @@ jobs:
           # >= 0.13.43 are built this way.
           for version in $(echo $RELEASES | sed -e 's/ /\n/g'); do
             git checkout "$version"
+            while ! nix-build -A tools -A cached nix; do; done
```

Comment:
Shall we put a limit to this? Right now, we're risking to get stuck in an infinite loop.